src/pepr/operator/controllers/config/config.ts

```diff
@@ -75,18 +76,18 @@ export async function updateCfgSecrets(cfg: kind.Secret) {
   }
 }
 
-export async function updateCfg(cfg: ClusterConfig) {
-  configLog.info("Updating UDS Config from uds-operator-config ClusterConfig change");
+async function handleCAUpdate(expose: ConfigExpose) {
+  if (!Object.keys(expose).includes("caCert")) {
+    return;
+  }
```

Comment:
Same consideration here, if someone removes their caCert later we would want to make sure it gets removed from the authservice config (treat this as `""`).